README.md

```diff
@@ -377,7 +377,7 @@ After running `ilab config init` your directories will look like the following o
 | `~/.local/share/instructlab/checkpoints/`  | Contains the output of the training process.                                |
 | `~/.config/instructlab/config.yaml`        | Contains the `config.yaml` file |
 
-You can view your `config.yaml` file with the following command:
+You can view your `config.yaml` file with the following command (or use `-k` to show a specific config section, `-wc` to show without comments, both options can also be used together):
```

Comment:
This adds a lot of text for the user to parse through and I am not sure it's really needed.  If the user should use these options here, we should probably just update the command below.  Otherwise I think they can discover the options later.

If we do really want to talk about the options here, I would suggest:

```(use `-k` to show a specific config section and/or `-wc` to show without comments):```

Reply:
thanks, yeah, just try to provide a small tips...